### PR TITLE
AO3-6819 Invalid HTML for Blocked Users and Muted Users pages

### PR DESCRIPTION
**Pull request by Dev21Maker**
# Pull Request Checklist

* [ ] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [ ] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [ ] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [ ] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [ ] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6819
## Purpose

Extracting headline from the list and removing the list's role

## Testing Instructions
Please refer to the testing instructions in the JIRA ticket

If you have a Jira account with access, please update or comment on the issue
with any new or missing testing instructions instead.

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

Dev21Maker (he/him)

What name and pronouns should we use to credit you in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?

If you have a Jira account, please include the same name in the "Full name"
field on your Jira profile, so we can assign you the issues you're working on.

*Please note that if you do not fill in this section, we will use your GitHub account name and
they/them pronouns.*

### CODE REVIEW

**Comment by sarken** [comment]
Hi, Dev21Maker!

Thank you so much for this pull request. Someone will be along to review it soon.

I've updated the Jira issue status to In Review, so no one will mistakenly create a duplicate pull request. If you'd like the ability to comment on, assign, and transition issues in the future, you're welcome to create a Jira account! You can just reply here with the account name and we'll set up the permissions for you. (It makes things a bit easier for us on the organizational side if the Full Name on your Jira account either closely matches the name you'd like us to credit in the release notes or includes it in parentheses, e.g. "Nickname (CREDIT NAME).")

Thanks again for contributing! If you have any questions, you can contact us at otw-coders@transformativeworks.org.

**Comment by Dev21Maker** [comment]
Hey @sarken, I've joined the project on Jira and the name should also be Dev21Maker, if you can't find me let me know

**Comment by sarken** [comment]
@Dev21Maker Found you! 😆 I've assigned the issue to you and updated your permissions.

The code change looks good! The only additional change this needs is the style fixes Reviewdog mentioned, but no rush. We're in a code freeze right now while we work on an internal project, so it will be a few weeks before we consider anything for merging. (You're still welcome to submit pull requests during the freeze, though!)

File: app/views/blocked/users/index.html.erb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <h2 class="heading"><%= t(".title") %></h2>
 
 <!--subnav-->
-<ul class="navigation actions" role="navigation">
-  <h3 class="landmark heading"><%= t("a11y.navigation") %></h3>
+<h3 class="landmark heading"><%= t("a11y.navigation") %></h3>
+<ul class="navigation actions">
```
Comment (brianjaustin) [inline app/views/blocked/users/index.html.erb:5]:
```suggestion
<ul class="navigation actions">
```

File: app/views/muted/users/index.html.erb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <h2 class="heading"><%= t(".title") %></h2>
 
 <!--subnav-->
-<ul class="navigation actions" role="navigation">
-  <h3 class="landmark heading"><%= t("a11y.navigation") %></h3>
+<h3 class="landmark heading"><%= t("a11y.navigation") %></h3>
+<ul class="navigation actions">
```
Comment (brianjaustin) [inline app/views/muted/users/index.html.erb:5]:
```suggestion
<ul class="navigation actions">
```

**Review by brianjaustin — CHANGES_REQUESTED** [review]
Thank you for the PR! Please address the ERB Lint failures (I've added what it seems to want as suggestions, since ERB Lint's messages are often rather… enigmatic), otherwise looks good to me

**Review by brianjaustin — APPROVED** [review]
Thank you!

**Pull request merged**